config/initializers/omniauth.rb

```diff
@@ -0,0 +1,14 @@
+OmniAuth.config.logger = Rails.logger
+
+Rails.application.config.middleware.use OmniAuth::Builder do
+  provider(
+    :google_oauth2, 
+    Rails.application.credentials.google_auth[:client_id], 
+    Rails.application.credentials.google_auth[:client_secret], 
+    { 
+      client_options: { 
+        ssl: { ca_file: Rails.root.join("cacert.pem").to_s } 
```

Comment:
Does this cert exist or will it be added in a separate PR/manual process?

Reply:
ha, copy pasted from a site, so weird, it was actually working and this file doesn't exist at all. Maybe a prod only type of thing that omniauth is just automagically handling?